core/src/main/java/com/linecorp/armeria/internal/common/ArmeriaHttpUtil.java

```diff
@@ -881,11 +881,11 @@ private static void toNettyHttp2Client(HttpHeaders inputHeaders, Http2Headers ou
         for (Entry<AsciiString, String> entry : inputHeaders) {
             final AsciiString name = entry.getKey();
             final String value = entry.getValue();
-            if (HTTP_TO_HTTP2_HEADER_BLACKLIST.contains(name)) {
+            if (HTTP_TO_HTTP2_HEADER_BLOCKLIST.contains(name)) {
                 continue;
             }
 
-            if (isTrailer && isTrailerBlacklisted(name)) {
+            if (isTrailer && isTrailerBlocklisted(name)) {
```

Comment:
Blocklisted -> Disallowed?

Reply:
Forbidden also sounds good, but I'm not sure if it's too "strict".

Reply:
That sounds good to me as well.